docs/design/README.md

```diff
@@ -2226,18 +2226,21 @@ being passed as a separate explicit argument.
 > -   Proposal
 >     [#950: Generic details 6: remove facets](https://github.com/carbon-language/carbon-lang/pull/950)
 
-### Checked and template parameters
+### Checked and unchecked parameters
 
-The `:!` indicates that `T` is a _checked_ parameter passed at compile time.
-"Checked" here means that the body of `Min` is type checked when the function is
+The `:!` indicates that `T` is a type that is bound at compile time. There are
+two forms of parameters: checked and unchecked. A parameter declared with the
+`template` keyword is an _unchecked parameter_, a parameter declared without the
+`template` keyword is a _checked parameter_. 
+
+"Checked" here means that the body of `Min` (above) is type checked when the function is
 defined, independent of the specific type values `T` is instantiated with, and
 name lookup is delegated to the constraint on `T` (`Ordered` in this case). This
 type checking is equivalent to saying the function would pass type checking
-given any type `T` that implements the `Ordered` interface. Then calls to `Min`
+given any type `T` that implements the `Ordered` interface. Subsequent calls to `Min`
```

Comment:
Why this change? Is there something ambiguous about the text before?

Reply:
@jonmeow Could you chime in here, do we have any style guidance for "then"?

Reply:
"Then" should typically be used to refer to a direct action sequence, for example:

First, we create an object. Then, we type check it.

"Subsequent" is closer to "Afterwards" in its sense, indicating a time ordering but not an immediate sequence.

https://diffsense.com/diff/subsequently/then

Here, "then" isn't used to refer to a specific consecutive action; instead, it's referring to potential future actions (and in plural). In that context, I do agree "Subsequent" is a better choice.